plugins/Phone_fr.py

```diff
@@ -117,6 +118,7 @@ class father:
                             (u"+33  3631", u"3631"),
                             (u"0102030405", u"+33 102030405"),
                             (u"01 02 03 04 05", u"+33 1 02 03 04 05"),
+                            (u"01.02.03.04.05", u"+33 1 02 03 04 05"),
```

Comment:
Please, keep input format. It's not really a fault, data still usable. It will raise lot of not interesting issue.

Reply:
The OpenStreetMap wiki says the phone fields should follow ITU-T E.123 which does not allow dots in the phone numbers. (*)

Also some Android phones don't work if there are dots in the phone number (I had the case with a Chinese vanilla Android 5 device). And on my device (Samsung Android 8), the Samsung GUI removes all dots as soon as you start editing a phone number. Maybe because the software author is a control freak who does not like dots, or more likely (imho) because they had reports of people putting dots in their phone numbers and running into trouble with one device or another (the contact list is shared across devices after all).

So I'd say dots should really go.


(*) The other references (ITU-T E.164, DIN 5008 and RFC 3966/NANP) don't say anything about formatting except RFC 3966 which says no spaces for phone URIs.

Reply:
I know, but Osmose must be fault tolerant at some point.

By experience, you can't flood contributors with issues asking to change data with low value added.